repo/packages/N/nexus/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://www.sharelytics.de/assets/img/nexus/nexus_48.png",
+    "icon-medium": "http://www.sharelytics.de/assets/img/nexus/nexus_96.png",
+    "icon-large": "http://www.sharelytics.de/assets/img/nexus/nexus_256.png"
```

Comment:
> please use assets backed by a cdn. maybe upload these images to your s3 bucket and use the public link from there?

We are working internally to get these available on a CDN. Will update when ready.